hr_attendance_modification_tracking/models/hr_attendance.py

```diff
@@ -17,3 +18,29 @@ class HrAttendance(models.Model):
     check_out = fields.Datetime(
         track_visibility='onchange'
     )
+    time_changed_manually = fields.Boolean(
+        string="Time changed",
+        required=False,
+        compute="_compute_time_changed_manually",
+        default=False,
+        store=True
+    )
+
+    @api.one
+    @api.depends('check_in','check_out')
+    def _compute_time_changed_manually(self):
+        if len(self.message_ids) > 3 and not self.time_changed_manually:
```

Comment:
Why 3 messages? On my point of view this is crazy, because if you write something it will be automatically changed... Also, If you create a new record it should be time_changed

Reply:
Outdated. 
We will apply many strategy changes in next commit.
Thanks.